tests/test_extension.py

```diff
@@ -0,0 +1,35 @@
+from unittest import mock
+import pytest
+
+from scrapy.crawler import Crawler
+from scrapy import Spider
+from spidermon.contrib.scrapy.extensions import Spidermon
+
+
+@pytest.fixture
+def spidermon_enabled_settings():
+    return {
+        'SPIDERMON_ENABLED': True,
+        'EXTENSIONS': {
+            'spidermon.contrib.scrapy.extensions.Spidermon': 500,
+        }
+    }
+
+
+@pytest.fixture
+def crawler(spidermon_enabled_settings):
+    crawler = Crawler(Spider, settings=spidermon_enabled_settings)
+    crawler.spider = Spider('dummy')
+    return crawler
+
+
+def test_engine_stopped_suites_should_run(crawler):
+    "The suites defined at engine_stopped_suites should be loaded and run """
+    engine_stopped_suites = [
+        'tests.fixtures.suites.Suite01'
+    ]
+    spidermon = Spidermon(crawler, engine_stopped_suites=engine_stopped_suites)
+    spidermon.engine_stopped_suites[0].run = mock.MagicMock()
+    spidermon.engine_stopped()
+    assert spidermon.engine_stopped_suites[0].__class__.__name__ == 'Suite01'
+    assert spidermon.engine_stopped_suites[0].run.called
```

Comment:
I'm fine using `mock` and installing through `requirements`.
Here, probably I would assign `mock` to it's own variable to make the assertions shorter.
Also, probably would use `my_mock.assert_called_once()` or other assertion built-in with `mock`.

Reply:
@ejulio I tried to use `assert_called_once()` but it is not supported in python 3.5. I went back from Raphael's approach.

Reply:
Are you sure?https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once

Reply:
@ejulio I replied on the wrong place, pasting here too:

https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.assert_called_once
`New in version 3.6.`

It was failing on python 3.5 build on Travis.

Reply:
My bad!
But, there are alternatives like `assert_any_call` or `assert_has_calls(1)`

Reply:
@ejulio I ended up using assert_called_once_with(mock.ANY), what do you think?

Reply:
Looks good to me.
Not sure what is better, if `assert_any_call` or `assert_called_once_with(mock.ANY)`.